src/providers/drafts.ts

```diff
@@ -0,0 +1,123 @@
+import { Language, License, VideoCategory } from '@/api/queries'
+import { CommonStore, createStore } from '@/store'
+import { createId } from '@/utils/createId'
+import { readFromLocalStorage } from '@/utils/localStorage'
+
+export type CommonDraftProps = {
+  id: string
+  channelId: string
+  updatedAt: string
+  seen?: boolean
+}
+
+export type Draft = VideoDraft
+
+export type DraftType = 'video'
+
+export type VideoDraft = {
+  type: 'video'
+  title?: string | null
+  description?: string | null
+  isPublic?: boolean | null
+  publishedBeforeJoystream?: string | null
+  hasMarketing?: boolean | null
+  isExplicit?: boolean | null
+  language?: Language['iso'] | null
+  category?: VideoCategory['id'] | null
+  licenseCode?: License['code'] | null
+  licenseCustomText?: License['customText'] | null
+  licenseAttribution?: License['attribution'] | null
+} & CommonDraftProps
+
+export type RawDraft = Omit<Draft, 'id' | 'updatedAt'>
+
+type DraftStoreStateV1 = {
```

Comment:
Let's just call it `DraftStoreState`, it's the only version of that store so far